codegen/templates/endpoint.tmpl

```diff
@@ -171,6 +171,8 @@ func (handler *{{$handlerName}}) HandleRequest(
 
 	{{if eq .ResponseType "" -}}
 	res.WriteJSONBytes({{.OKStatusCode.Code}}, cliRespHeaders, nil)
+	{{- else if eq .ResponseType "string" -}}
+	res.WriteJSONBytes({{.OKStatusCode.Code}}, cliRespHeaders, []byte(response))
```

Comment:
I don't think this is valid json ? `"\"" + response + "\""` ?

Reply:
the response is a string itself. 
check out `build/endpoints/bar/bar_bar_method_helloworld.go`